backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -866,6 +867,12 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
       checkOperatorMatch[ProjectExecTransformer])
   }
 
+  test("test 'scala udf'") {
```

Comment:
Add `ENABLE_UDF_EXAMPLE` into CMake, and it is ON  by default for Spark UT

Reply:
option ENABLE_LOCAL_UDFS is added, because the number of udf subdirectories maybe more than 1. ENABLE_LOCAL_UDFS is used to control whether those udf source files need to be compiled.